domain/model/service/service.go

```diff
@@ -242,6 +242,13 @@ func (s *Service) CheckModelExists(ctx context.Context, modelUUID coremodel.UUID
 	return s.st.CheckModelExists(ctx, modelUUID)
 }
 
+// ModelRedirection returns redirection information for the current model. If it
+// is not redirected, [modelmigrationerrors.ModelNotRedirected] is returned.
+// Placeholder for model migration.
+func (s *Service) ModelRedirection(ctx context.Context, modelUUID coremodel.UUID) (model.ModelRedirection, error) {
+	return model.ModelRedirection{}, modelerrors.ModelNotRedirected
+}
```

Comment:
This is interesting, as we could end up with unbounded growth on this. Although, I can't think of anyway to limit it without causing issues for CMR.

I think we could just add to it, but add a note that we might want to prune this somehow in the future.

Reply:
This is OK. This unbounded growth in this case is:
1. Tied to an explicit action.
2. Is reversible.
3. Only done by authorised users.
4. Is inherently useful.